addons/website/controllers/form.py

```diff
@@ -175,7 +175,10 @@ def extract_data(self, model, values):
             # If it's a known field
             elif field_name in authorized_fields:
                 try:
-                    input_filter = self._input_filters[authorized_fields[field_name]['type']]
+                    field_type = authorized_fields[field_name]['type']
+                    if field_type in ['char', 'text', 'html']:
+                        field_value = html_escape(field_value)
```

Comment:
Isn't there something less drasctic than `html_escape`?
If one is typing `1 < 2` in a question field he will end up with `1 &lt; 2`. Same with `Me & You` -> `Me &amp; You`
It's just one case among others but you got the idea. Have you considered it?

Reply:
Suggestion 1: try to parse as HTML, if successful, check if there are several nodes, and only escape in that case.
Suggestion 2: do not fix on that layer, but instead escape when putting the values in an email.

I'd go for (2) because the issue is related to the fact the email message is in HTML.

Reply:
@rdeodoo 
(2) was a dead end because values are already stored into a single text at that point.

I did the following:
- for known fields, only apply to `html` fields (not all text fields)
- for both known `html` fields and unknown/custom fields, `html_escape` only if text contains `<` and `>`.